normalize_repos/normalize_repos.py

```diff
@@ -18,6 +18,9 @@
 from set_labels import set_labels
 import branch_protections
 
+# For converting .cc-metadata.yml to Python dictionary
+import yaml
```

Comment:
You might have to install [PyYAML](https://pypi.org/project/PyYAML/) using [Pipenv](https://pipenv.pypa.io/en/latest/).

Reply:
Yes, I activated the pipenv shell and then installed it via pip.

Reply:
You should install it directly using Pipenv.
```bash
$ pipenv install pyyaml
```
This will add the dependency in `Pipfile` and `Pipfile.lock`.

Reply:
Understood.